skyportal/handlers/api/sources_in_gcn.py

```diff
@@ -0,0 +1,213 @@
+from baselayer.app.access import auth_or_token, permissions
+from skyportal.models.gcn import GcnEvent
+from ..base import BaseHandler
+from ...models import Localization, SourcesInGCN
+
+from .source import get_sources, MAX_SOURCES_PER_PAGE
+import sqlalchemy as sa
+
+
+class SourcesInGcnHandler(BaseHandler):
+    @permissions(['Manage GCNs'])
+    def post(self, dateobs):
+        data = self.get_json()
+        if 'localizationName' not in data:
+            return self.error("Missing required parameter: localization_name")
+        if 'localizationCumprob' not in data:
+            return self.error("Missing required parameter: localization_cumprob")
+        if 'sourceId' not in data:
+            return self.error("Missing source_id")
+        if 'confirmedOrRejected' not in data:
+            return self.error("Missing confirmed or rejected")
```

Comment:
The issue i have here is that this table keeps track of sources that are confirmed or rejected, but not the sources that are simply 'undefined' (in the localization, obtained with a query on the sources on the fly, those are not stored in the DB as you know). 
So the true or false of this table means that it is confirmed or rejected. In my mind it was 'lighter' in the table to use a boolean to reflect these 2 possible states, rather than a string.
The issue with the second option you give here is that then I need to add 2 columns, one being a boolean for confirmed, and the second for reject. 

Should I just go for a string 'confirmed' or 'rejected' rather than the boolean ?

Reply:
I think it is fine to have one boolean column called "confirmed". If it is True, it is confirmed, if it is False it is rejected, and if the row doesn't exist then it is neither (in a different scenario you can imagine having a nullable boolean column that does the same). 
The fact that you have a third option should be explained in the docstring for this column (and maybe in other key places) for the developer to understand the subtlety of it... but other than that I think it is fine to have it as a boolean.